iOSiTunesSearch/iOSiTunesSearch/Controllers/SearchResultController.swift

```diff
@@ -0,0 +1,76 @@
+import Foundation
+
+class SearchResultController: Codable {
+    
+    static let endpoint = "https://itunes.apple.com/search?"
```

Comment:
The question mark here should be added by the URLComponents object, so that the format is correct. As it is now, it isn't actually affecting the final url, but it's common practice to let URLComponents add that to your URL for you.